README.md

```diff
@@ -79,4 +79,4 @@ If you think this project is worth supporting financially please make a contribu
 
 ## Things that are currently annoying me
 
-[![](https://codescene.io/projects/697/status.svg) Get more details at **codescene.io**.](https://codescene.io/projects/697/jobs/latest-successful/results)
\ No newline at end of file
+[![](https://codescene.io/projects/697/status.svg) Get more details at **codescene.io**.](https://codescene.io/projects/697/jobs/latest-successful/results)

```

Comment:
This badge is outdated. So, it should be updated!

[![](https://codescene.io/projects/697/status.svg) Get more details at **codescene.io**.](https://codescene.io/projects/697/jobs/latest-successful/results)
Hmm... It looks strange... No badge!

Taking into account these Blame results: [Blame README.md](https://github.com/ThreeMammals/Ocelot/blame/develop/README.md), I would say that markdown code of the badge is outdated.
Check the latest docs regarding Badges: [Keep Tabs on the State of your Code with Badges — CodeScene 1 Documentation](https://codescene.io/docs/integrations/publish-status-badges.html)
I've double checked, and, Yes, current markdown is invalid!

Valid markdown of the Health Badge :point_down: 
```markdown
[![Ocelot Code Health](https://codescene.io/projects/697/status-badges/code-health)](https://codescene.io/projects/697)
```
#### Example of my project badge
[![CodeScene Code Health](https://codescene.io/projects/38939/status-badges/code-health)](https://codescene.io/projects/38939)

Finally, you need to change line 82 with this code:
```markdown
[![Ocelot Code Health](https://codescene.io/projects/697/status-badges/code-health)](https://codescene.io/projects/697) Get more details at [**codescene.io**](https://codescene.io/projects/697/jobs/latest-successful/results)
```
So, it will look like:
> [![Ocelot Code Health](https://codescene.io/projects/697/status-badges/code-health)](https://codescene.io/projects/697) Get more details at [**codescene.io**](https://codescene.io/projects/697/jobs/latest-successful/results)

Note, we don't see the badge because project settings is wrong. We need to ask Tom to fix project settings. It seems the badge is disabled.

Reply:
Hey, I opened this ages ago.. Sorry, won't touch it anymore. Don't even know what the badge is about.

Reply:
Hey, I opened this ages ago.. Sorry, won't touch it anymore. Don't even know what the badge is about.

Reply:
> Hey, I opened this ages ago.. Sorry, won't touch it anymore. Don't even know what the badge is about.

Not ages, but 6 months ago! :smile_cat: 
:ok_hand:  Won't you collaborate anymore, Leonardo?

Reply:
Fine. Re-read the PR. New conclusion:
I'm sorry, but my PR was about typos. What you are requesting is beyond the scope of this PR. My only change on line 82 was adding a '['